cmd/up/exec.go

```diff
@@ -397,15 +402,19 @@ func (up *upContext) RunCommand(ctx context.Context, cmd []string) error {
 		return err
 	}
 
+	k8sClient, restConfig, err := up.K8sClientProvider.Provide(okteto.Context().Cfg)
```

Comment:
I can't test this path if I don't create a new state file controller. Let me know what you think about creating it